nvflare/edge/tools/edge_recipe.py

```diff
@@ -59,11 +106,27 @@ def __init__(
     ):
         self.device_selection_size = device_selection_size
         self.min_hole_to_fill = min_hole_to_fill
+        # check if min_hole_to_fill is smaller than device_selection_size
+        if min_hole_to_fill > device_selection_size:
+            raise ValueError("min_hole_to_fill needs to be smaller than device_selection_size")
```

Comment:
good check suggest msg:

"min_hole_to_fill needs to be smaller than or equal to device_selection_size"

Reply:
right, updated